core/templates/dev/head/filters.js

```diff
@@ -305,7 +305,7 @@ oppia.filter('normalizeWhitespacePunctuationAndCase', [function() {
 oppia.filter('convertToPlainText', [function() {
   return function(input) {
     var strippedText = input.replace(/(<([^>]+)>)/ig, '');
-    strippedText = strippedText.replace('&nbsp;', ' ');
+    strippedText = strippedText.replace(/&nbsp;/ig, '');
```

Comment:
Just to check, should the second arg here be a space rather than an empty string?

Reply:
It hardly matters because even if I use a space the CSS class would collapse it. But still, I'll do it.